templates/includes/sidebar.html

```diff
@@ -0,0 +1,18 @@
+{% extends "base.html" %}
+
+{% block sidebar %}
+    <ul>
+        <li>
+            <a href="#">Home page</a>
```

Comment:
Instead of a hardcoded link, it would be better to use the Django `{% url 'url-name' %}` tag. This way, if the URL changes in the future, the link will automatically update. You just need to replace `'url-name'` with the name of the URL you want to link to.